source/common/network/cidr_range.cc

```diff
@@ -150,7 +150,7 @@ InstanceConstSharedPtr CidrRange::truncateIpAddressAndLength(InstanceConstShared
       return address;
     } else if (length == 0) {
       // Create an Ipv4Instance with only a port, which will thus have the any address.
-      return std::make_shared<Ipv4Instance>(uint32_t(0));
+      return *createInstance<Ipv4Instance>(uint32_t(0));
```

Comment:
change this method to return the StatusOr and check results up stack. Here and below.